src/models/treemodel.cpp

```diff
@@ -44,6 +44,9 @@ QVariant BottomUpModel::headerColumnData(int column, int role) const
         case Binary:
             return tr("Binary");
         }
+        if (m_diffMode && column - NUM_BASE_COLUMNS == 1) {
```

Comment:
this doesn't work correctly for perf data files with more than one cost, or? i.e. something like `perf record --events cycles,instructions`. further down below you use modulus - I think you'd need the same here.